.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: ZvGUYKfixOzJWtTHwR9qWuuUwhvY6R9k5
```

Comment:
A few questions

1) Is this file necessary?
2) Is this sensitive information that should be excluded from source?
3) If the answer to question two is yes, how would you recommend proceeding?

Reply:
Thanks @SherSpock I was following [this blog](https://coveralls.io/github/AgileVentures/sfn-client) to get this setup and there is a suggestion its needed if we are not using travis pro. Please see below
<img width="776" alt="Screen Shot 2019-05-10 at 11 29 46 AM" src="https://user-images.githubusercontent.com/1543546/57514401-1b3efc00-7319-11e9-9c71-ec22425439ce.png">
Maybe we can pass this from a .env though I have seen it added to other repos like the wso-fe so maybe its not that sensitive.

Reply:
The link to blog post you posted goes [here](https://coveralls.io/github/AgileVentures/sfn-client) so I wasn't able to see what information you were using for this work.

Some relevant links:

https://stackoverflow.com/questions/39501417/how-can-i-connect-coveralls-and-travis-in-github
https://github.com/tapjs/node-tap/issues/210

Personally my feeling is the token should be removed from source, then rotated. Its presence on wso-fe is likely a mistake. In general any creds-like object in source should be looked at with suspicion. 

Still open to counterarguments on this specific instance.

Reply:
Thank you @SherSpock I have deleted the file `.coveralls.yml` as its not needed. See [this](https://github.com/lemurheavy/coveralls-public/issues/208) as the motivating factor.